### ISSUE TITLE
Javascript Error on Preview Page

### ISSUE THREAD

**Issue by hannemann** [Laravel-Backpack/CRUD#1792]
# Bug report

### What I did:
Added Preview

### What I expected to happen:
Show contents without any Errors

### What happened:
Javascript Error: Uncaught ReferenceError: $ is not defined

### What I've already tried to fix it:
surround script tags in:

vendor/backpack/crud/src/resources/views/buttons/clone.blade.php
vendor/backpack/crud/src/resources/views/buttons/delete.blade.php

with:

@push('after_scripts')
...
@endpush

This fixes the error and i think it does not break any functionality
Error occurs because jQuery is referenced before it is loaded

### Backpack, Laravel, PHP, DB version:
Backpack CRUD 3.5, Laravel 5.7

**Comment by hannemann** [comment]
I was too confident about my idea on how to fix it :(

After cloning of something with above fix applied the database has a new entry but the global crud object which is utilized to reload the page is undefined.

crud.table.ajax.reload();

Uncaught ReferenceError: crud is not defined

**Comment by hannemann** [comment]
To make it work on both pages list and preview i ended up asking if the request is ajax before pushing the script away...

@if (!request()->ajax())
	@push('after_scripts')
@endif
<script>...</script>
@if (!request()->ajax())
	@endpush
@endif

and asked for crud before trying to reload the page.

**Comment by pxpm** [comment]
I found the same issue on preview page:

```
Uncaught ReferenceError: $ is not defined
    at 1:309
```

Simple preview page with only one text column.

**Comment by tabacitu** [comment]
Hi guys,

Thanks for the heads-up, and the solution. @hannemann I've found you're right about the source of the problem (jQuery not being loaded), but only found it's a problem for the CLONE button, since DELETE shouldn't even be loaded on the Show page.

I've pushed a fix here for the Clone button - https://github.com/Laravel-Backpack/CRUD/commit/cd526f0e1ae9293911abb75690183846ea3f50bb , which will be tagged and released later today. Let me know if this doesn't fix it for you entirely.

Cheers!
